app/assets/javascript/application.js

```diff
@@ -9,6 +9,33 @@ jQuery(function($) {
   });
 
   ////
+  // Add "select all"/"clear all" buttons to each select2 dropdown menu
+  $("select.select2").each(function() {
```

Comment:
Would be good to pull this out into a `GOVUK` module. Perhaps we can even pass put it into `govuk_admin_template`.